src/constructors.jl

```diff
@@ -8,12 +8,14 @@ Arf(x; prec::Integer = _precision(x)) = set!(Arf(prec = prec), x)
 # disambiguation
 Arf(x::Arf; prec::Integer = precision(x)) = set!(Arf(prec = prec), x)
 Arf(x::Rational; prec::Integer = _precision(x)) = set!(Arf(prec = prec), x)
+Arf(x::BigInt; prec::Integer = precision(Arf)) = set_mpz!(Arf(prec = prec), x)
 
 #Arb
 Arb(x; prec::Integer = _precision(x)) = set!(Arb(prec = prec), x)
 # disambiguation
 Arb(x::Arb; prec::Integer = precision(x)) = set!(Arb(prec = prec), x)
 Arb(x::Rational; prec::Integer = _precision(x)) = set!(Arb(prec = prec), x)
+Arb(x::BigInt; prec::Integer = precision(Arb)) = Arb(Arf(x; prec = prec); prec = prec)
```

Comment:
I'd rather have to as
```julia
set!(Arb(prec = prec), Arf(x, prec = prec))
```
to keep in line of using `set!`-ters. I know this is not exactly the julia way, but that's what it is on the `arb` level